pkg/configs/db/postgres/postgres.go

```diff
@@ -104,7 +105,7 @@ func (d DB) findConfigs(filter squirrel.Sqlizer) (map[string]configs.View, error
 		var cfg configs.View
 		var cfgBytes []byte
 		var userID string
-		err = rows.Scan(&cfg.ID, &userID, &cfgBytes)
+		err = rows.Scan(&cfg.ID, &userID, &cfgBytes, &cfg.DeletedAt)
```

Comment:
This `cfg.DeletedAt` has to be time or null to be able to store null time from DB.